lib/src/Uncertainty/Distribution/Normal.cxx

```diff
@@ -709,7 +709,22 @@ Distribution Normal::getStandardRepresentative() const
 Scalar Normal::getRoughness() const
 {
   // 0.2820947917738781434740398 = 1 / (2 * sqrt(Pi))
-  return 0.2820947917738781434740398 / getSigma()[0];
+  if (dimension_ == 1)
+    return 0.2820947917738781434740398 / getSigma()[0];
+
+  Scalar roughness = 1.0;
+  if (hasIndependentCopula())
+  {
+    for (UnsignedInteger d = 0; d < dimension_; ++d)
+      roughness *= 0.2820947917738781434740398 / getSigma()[d];
+    return roughness;
+  }
+  else
+  {
+    for (UnsignedInteger d = 0; d < dimension_; ++d)
+      roughness *= 0.2820947917738781434740398 / cholesky_(d, d);
```

Comment:
This sounds much less obvious for me, because I do not see clearly why the off-the-diagonal terms do not impact the result. Did you find a theoretical reference or did a personal calculation?

Reply:
@mbaudin47 you just need to write what is the `roughness`, ie \int density^2(x) dx

Reply:
![calcul_roughness](https://user-images.githubusercontent.com/1433220/82088457-14985100-96f2-11ea-9104-f64de162836e.png)